<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCViewer</class>
 <widget class="QMainWindow" name="QCViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCViewer</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="mainView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="interactive">
         <bool>false</bool>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorViewCenter</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::MinimalViewportUpdate</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="documentTitle">
         <string notr="true"/>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuArchitecture">
    <property name="title">
     <string>Architecture</string>
    </property>
    <widget class="QMenu" name="menuType">
     <property name="title">
      <string>Type</string>
     </property>
     <addaction name="actionComplete"/>
     <addaction name="actionLinear_nearest_neighbour"/>
     <addaction name="actionCircular_nearest_neighbour"/>
     <addaction name="separator"/>
     <addaction name="actionCustom"/>
    </widget>
    <addaction name="actionDraw_warnings"/>
    <addaction name="separator"/>
    <addaction name="menuType"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuArchitecture"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="fileOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLinear_nearest_neighbour">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Linear Nearest Neighbour</string>
   </property>
  </action>
  <action name="actionCircular_nearest_neighbour">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Circular Nearest Neighbour</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Custom...</string>
   </property>
  </action>
  <action name="actionDraw_warnings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw warnings</string>
   </property>
  </action>
  <action name="actionComplete">
   <property name="text">
    <string>Complete</string>
   </property>
  </action>
  <action name="parseCommandLine">
   <property name="text">
    <string>parseCommandLine</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>parseCommandLine</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
